src/requirements/Allowlist/AllowlistRequirement.tsx

```diff
@@ -24,10 +25,14 @@ const AllowlistRequirement = ({ ...rest }: RequirementProps): JSX.Element => {
     <Requirement
       image={<Icon as={ListPlus} boxSize={6} />}
       footer={
-        hideAllowlist && (
-          <Text color="gray" fontSize="xs" fontWeight="normal">
-            {`Allowlisted ${isEmail ? " email" : ""} addresses are hidden`}
-          </Text>
+        isEmail ? (
```

Comment:
I this way the "allowlisted ... addresses are hidden" test wont show up, is that intended? 👀

Reply:
Oh, right, changed in https://github.com/guildxyz/guild.xyz/pull/1140/commits/95dc304384882c6cefe24fc851b1cb3ab5bd2c70 👍